cf-agent/verify_processes.c

```diff
@@ -231,12 +233,29 @@ static PromiseResult VerifyProcessOp(EvalContext *ctx, const Attributes *a, cons
             {
                 if (IsExecutable(CommandArg0(a->process_stop)))
                 {
-                    ShellCommandReturnsZero(a->process_stop, SHELL_TYPE_NONE);
+                    Log(LOG_LEVEL_DEBUG, "Found process_stop command '%s' is executable.", a->process_stop);
+
+                  if (ShellCommandReturnsZero(a->process_stop, SHELL_TYPE_NONE))
```

Comment:
Wrong indentation here.

Reply:
Finally got cfengine-code-style loaded in emacs, fixed.